### ISSUE TITLE
We don't know what signs we need up in the space

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#20]
Since the "Day of the Long Poster-rippers" ;-) we've needed new and better signs up round the space.

I've made a start, with the process detailed in [the Style Guide](https://github.com/DoESLiverpool/somebody-should/wiki/Style-Guide-(signs,-logo,-etc.)) but this issue is to gather a list of signs we need, so we can work through making them.

Rectangular:
- [ ] WiFi signs for the Workshop x 2
- [ ] "Don't Panic" phone numbers sign x 3
- [ ] Opening times on the main door ("9.30 am to 5.30pm, Monday to Friday")
- [ ] Please wash your own dishes. (or similar wording, for kitchen)
- [ ] Please tidy up after yourself. (or similar wording, for main room)
- [ ] CCTV sign (@johnmckerrell has a not very good one)
- [ ] Explanation of how to store things (box to use, pricing, etc.) in the storage area
- [ ] Explanation of what to do when recycling boxes are full
- [ ] Sign asking people not to take photos of other people or projects without permission

Vinyl-cut:
- [x] "Wait X minutes before opening" sign for Gerald
- [x] 2 x "Do not cut metal" signs for both Gerald and Sophia

Diamond:
- [ ] "Food Sink" sign above the food sink
- [ ] "Workshop Sink" sign above the workshop sink

Door sign:
- [ ] Workshop sign for the workshop door
- [ ] Interests for the "physical social network" wall in the main room
  - [ ] Laser-cutting
  - [ ] Languages (non-programming)
  - [ ] Coding
  - [ ] 3d printing

Circular Warning:
- [x] "Rule Zero: Do not be on Fire" for the Workshop door

Diamond Warning:
- [ ] RepRap 3D Printer (the UPs have stickers): Hot warning
- [ ] CNC mill: 
  - [x] Eye and ear protection
  - [ ] Finger injury warning
- [ ] Band Saw:
  - [ ] Eye and ear protection
  - [ ] Finger injury warning
- [ ] Soldering
  - [ ] Fume warning
  - [ ] Hot warning
  - [ ] Lead warning
  - [ ] Wash hands after use

Vinyl:
- [ ] "Max Power: 50" for Sophia
- [ ] "Max Power: 70" for Gerald

(Tick the above signs off once done, and add new items to the list if you think of more - edit this info to do that)

**Comment by johnmckerrell** [comment]
Just going to leave this as a comment for now, a poster advertising this issue list would be good.

I have an idea that would be wartime propaganda style. On the left it says "See an issue? Report it!" on the right "Got 5 minutes? Help us out!". Across the bottom would be the link to this github issues link.

**Comment by amcewen** [comment]
Ooh, that's a nice idea.  Would need the right sort of propaganda vibe, you're right.  Maybe see when we get through the rest of the ones on this issue...

**Comment by DoESsean** [comment]
As long as it's not in the style of 'Keep Calm...' posters, sounds good to
me.

**Comment by amcewen** [comment]
@DoESsean has bought some sheets of black A1 mountboard for signs. They're hidden away in the big white envelope behind the cupboard by my desk, for when anyone gets to making some signs.  And when that runs out, the mountboard came from the art shop on Slater Street.

**Comment by DefProc** [comment]
Ben did a load of signs that are now in dropbox (I think).

**Comment by johnmckerrell** [comment]
I've ordered some CCTV signs, went for a more subtle rectangular one rather than an obnoxious yellow diamond. It's actually one to be mounted in a window but maybe we can stick one to some clear acrylic by the lift too?

**Comment by johnmckerrell** [comment]
Sign recommending proper disposal of refuse, "please recycle all you can" "please dispose of refuse responsibly" "if a recycling bin is full, please empty and reset with a new bag"

**Comment by johnmckerrell** [comment]
We need a sign for what to do when the recycling bins are full (added above)

**Comment by johnmckerrell** [comment]
Occurred to me that we might get more hard-wearing signs by sublimating onto acrylic. I had originally thought you might still cut an acrylic border but maybe even that could be printed on. I wonder whether it would be easier to cut the shape (rounded corners) before or after printing, I'm guessing before. Obviously they're more expensive but if they last longer and don't start peeling away it could be worth it.

**Comment by jackie1050** [comment]
That seems like a good idea, although we would have to do some work on how they ought to look.
When I first read this I thought you meant for outside, and I think there might be some issues with lightfastness – some of the bracelets have been around for quite a while though, and haven’t noticably faded indoors.

**Comment by amcewen** [comment]
We should have a sign-making evening/day sometime to clear through this (as suggested by @goatchurchprime in #621).  One Saturday in January maybe (things are getting busy in the run up to Christmas now)?

It'd be good to decide on the general design principles beforehand, if we want to switch from the mountboard approach (documented in https://github.com/DoESLiverpool/somebody-should/wiki/Signs now).  @zarino maybe you and I (anyone else too?) could have a chat about that side sometime?

**Comment by Sean-anotherone** [comment]
A "standard" Health and Safety at Work Act poster is a mandatory requirement for any workplace - I don't recall seeing one anywhere ?

I'll ask in the print unit at the office if they have any knocking about.

**Comment by DoESsean** [comment]
Right, the H&S at Work poster isn't mandatory - it can be foregone if each employee (ie me) is given a copy of it in leaflet form. It's now in my drawer. It won't hurt to have one anyway, but legally we're good.

We need to have signs up saying about CCTV.

**Comment by RiverMersey** [comment]
Could this thread / issue be closed again now?

**Comment by DoESsean** [comment]
I'd say probably not. There's still a lot of signage that needs making, it's just finding people that have the know-how and time to do it.

There is part of me that wonders whether it is worth restarting these very old issues as new ones, bringing them back into people's consciousness a bit, and getting further input from people who may not venture this far down the list. However, I think that may well be counter productive, as there is a certain cache that comes with closing a very old issue, and closing them and reopening anew would take that away.

**Comment by johnmckerrell** [comment]
We might like to have a poster relating to photography, added the following above:
* Sign asking people not to take photos of other people or projects without permission

**Comment by seanspotatobusiness** [comment]
Regarding hard-wearing signs, laserable laminate might work: https://www.hindleys.com/laser-laserables-ii-engraving-laminate-sheet.html - I've never used it but want to, some day.
